CONTRIBUTING.md

```diff
@@ -8,31 +8,41 @@ Thanks for helping making KEDA better!
 You can easily release a new Helm chart version:
 
 1. Update the version of the Helm chart in `Chart.yaml`
-2. Package the Helm chart
+2. Package the Helm chart. For KEDA:
 ```shell
 $ helm package keda
 Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
 ```
 
-3. Move the new chart to the docs folder
+For HTTP Addon:
+
+```shell
+$ helm package http-add-on
+Successfully packaged chart and saved it to: /home/ecomaz/src/keda/charts/keda-add-ons-http-0.2.0.tgz
+```
+
+1. Move the new chart to the docs folder. For both KEDA and the HTTP Addon, use the below command:
```

Comment:
I would keep all steps simple and only list when changes are different based on the app that is being shipped:
```suggestion
1. Move the new chart to the docs folder.
```

It feels more redundant than it adds value - Thoughts?

Reply:
Yup, agreed. Fixed in https://github.com/kedacore/charts/pull/194/commits/edeb37316c6d93b7c5aa827a9cb45ea1d904679c